hc/api/tests/test_create_check.py

```diff
@@ -48,22 +51,25 @@ def test_it_works(self):
     def test_it_accepts_api_key_in_header(self):
         payload = json.dumps({"name": "Foo"})
 
-        ### Make the post request and get the response
-        r = {'status_code': 201} ### This is just a placeholder variable
+        # Make the post request and get the response
+        r = self.client.post(self.URL, payload,
+                             content_type="application/json",
+                             HTTP_X_API_KEY="abc")
 
-        self.assertEqual(r['status_code'], 201)
+        self.assertEqual(r.status_code, 201)
 
     def test_it_handles_missing_request_body(self):
-        ### Make the post request with a missing body and get the response
-        r = {'status_code': 400, 'error': "wrong api_key"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "wrong api_key")
+        # Make the post request with a missing body and get the response
+        r = self.client.post(self.URL, content_type="application/json")
+        self.assertEqual(r.status_code, 400)
+        self.assertEqual(r.json()["error"], "wrong api_key")
```

Comment:
For now i think we can do with many asserts but asserting 400 should be sufficient to show that it handles a missing body. i.e. bad request.

Reply:
@dondrzzy, I agree with you. I am working on it.